x-pack/plugins/threat_intelligence/common/types/indicator.ts

```diff
@@ -32,13 +32,12 @@ export enum RawIndicatorFieldId {
   FileSha3512 = 'threat.indicator.file.hash.sha3-512',
   FileSha512224 = 'threat.indicator.file.hash.sha512/224',
   FileSha512256 = 'threat.indicator.file.hash.sha512/256',
-  FileSSDeep = 'threat.indicator.file.ssdeep',
-  FileTlsh = 'threat.indicator.file.tlsh',
-  FileImpfuzzy = 'threat.indicator.file.impfuzzy',
-  FileImphash = 'threat.indicator.file.imphash',
-  FilePehash = 'threat.indicator.file.pehash',
-  FileVhash = 'threat.indicator.file.vhash',
-  FileTelfhash = 'threat.indicator.file.elf.telfhash',
+  FileSSDeep = 'threat.indicator.file.hash.ssdeep',
+  FileTlsh = 'threat.indicator.file.hash.tlsh',
+  FileImpfuzzy = 'threat.indicator.file.hash.impfuzzy',
+  FileImphash = 'threat.indicator.file.hash.imphash',
+  FilePehash = 'threat.indicator.file.hash.pehash',
+  FileVhash = 'threat.indicator.file.hash.vhash',
```

Comment:
modified these using [this source of truth](https://docs.google.com/spreadsheets/d/1oYx9CQodKVmr0OtlJcjm1mNoUBlDcbHai0UYxtP8YzY/edit#gid=1814426429)